models/admin.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
```

Comment:
You shouldnt have a Admin model because it duplicates data with User. Instead you should have a isAdmin column in your User table